### PR TITLE
Return DTOs instead of plain hashes

### PR DESCRIPTION
**Pull request by maksimf**
This PR adds DTOs: `Attribute`, `Environment`, `Identity`, `TrafficType`. Now calling, e.g. `Models::Attributes.save(data)` will return `DataObjects::Attribute`

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2.0.0
+
+Return DTOs instead of hashes
```
Comment (patricioe) [inline CHANGES]: Let's make it 1.1 please. Since it hasn't been used yet.

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,7 @@
+2.0.0
```
Comment (patricioe) [inline NEWS]: same

**Comment by maksimf** [comment]
@patricioe conflicts solved

**Comment by patricioe** [comment]
@maksimf merging this. Remember to issue PRs against `development` branch.

**Comment by maksimf** [comment]
@patricioe there is no development branch in this repo, do you want me to create one?

**Pull request merged**